doc/General/Acknowledgement.md

```diff
@@ -1,64 +0,0 @@
-source: General/Acknowledgement.md
```

Comment:
IMHO we should keep this out of respect for all the projects we rely on.

Reply:
It's outdated deluxe

Reply:
I've put it back in, it does need updating but I believe it's important we recognise other projects we rely on in the spirit of open source :)